frontend/src/components/cluster-dashboard/cluster-list-view/ClusterListView.tsx

```diff
@@ -0,0 +1,158 @@
+                    <span className="di-inline-blk">
+                      <FiberManualRecordIcon className={"member-status " + member.status.toLowerCase()}/>
+                    </span>
+                    <Tooltip title={member.node} placement="bottom">
+                      <span className="member-name di-inline-blk">{member.node.split('://')[1]}</span>
+                    </Tooltip>
+                    {
+                      member.node === clusterData.leader ? 
+                      <Tooltip title="Leader Node" placement="bottom">
+                        <span className="member-type di-inline-blk leader">L</span>
+                      </Tooltip>
+                      : ""
+                    }
+                    {
+                        member.node === clusterData.oldest ?
+                        <Tooltip title="Oldest Node" placement="bottom">
+                          <span className="member-type di-inline-blk oldest">O</span>
+                        </Tooltip>
+                        : ""
+                    }
+                  </div>
+                }>
+                  <TreeItem nodeId={index + "child"} label={
+                    <div className="member-details">
+                      <div><span className="fo-we-bo">node : </span><span>{member.node}</span></div>
+                      <div><span className="fo-we-bo">nodeUid : </span><span>{member.nodeUid}</span></div>
+                      <div><span className="fo-we-bo">status : </span><span>{member.status.toLowerCase()}</span></div>
+                      <div><span className="fo-we-bo">roles : </span><span>{member.roles.join(',')}</span></div>
+                      <div className="member-actions">
+                        <Button variant="contained" size="small" onClick={(): void  => handleMember(member.node, "Leave")}>LEAVE</Button>
+                        <Button className="themed" variant="contained" size="small" onClick={(): void  => handleMember(member.node, "Down")}>SHUTDOWN</Button>
+                      </div>
+                    </div>
+                  } />
+                </TreeItem>
+            }) : ""
+          }
+          </TreeView>
+        </div>
+        <div className="add-member-container">
+          <TextField value={addNodeAddress} onChange={(event: React.ChangeEvent<HTMLInputElement>): void => { setAddNodeAddress(event.currentTarget.value) }} 
+            label="Node Address" placeholder="Enter Node Address" margin="dense" variant="outlined" fullWidth={true}/>
+          <Tooltip title="Add Node" placement="bottom">
+            <Fab variant="extended" size="small" aria-label="add node address" onClick={(): void  => addMember()}>
+              <AddIcon/>
+            </Fab>
+          </Tooltip>
+        </div>
+      </Grid>
+      <SimpleSnackBar message={snackBarMessage} open={openSnackBar} setOpen={setOpenSnackBar}/>
+      <ConfirmationDialog title={confDialogTitle} content={confDialogContent} open={openConfDialog} 
```

Comment:
Instead of avoiding long line by doing this, have all your props on their own line instead. More readible imo

Reply:
Just setup my eslint to use "jsx-max-props-per-line". Two props per line is ideal imo. That way I can avoid smaller props taking up an entire line.

Reply:
Btw I appreciate all the detailed comments, Davis, thank you. Feel free to drop more comments! 💯